tests/benchmark/compute/instruction/test_system.py

```diff
@@ -276,6 +285,8 @@ def test_contract_calling_many_addresses(
             ),
         ),
     )
+    attack_block += Op.JUMPI(Op.GT(Op.GAS, loop_cost), destination)
+    attack_block += Op.RETURN(0, 0)
 
     benchmark_test(
         code_generator=JumpLoopGenerator(
```

Comment:
This is not really correct, we don't need a JumpLoopGenerator here anymore, since I added the loop earlier. I am thinking about how to make it cleaner